core/encoding/uuid/reading.odin

```diff
@@ -151,16 +152,31 @@ time_v1 :: proc "contextless" (id: Identifier) -> (timestamp: u64) {
 	return cast(u64)transmute(u64le)timestamp_octets
 }
 
+
 /*
-Get the timestamp of a version 6 UUID.
+Get the timestamp of a version 1 UUID.
+
+Inputs:
+- id: The identifier.
+
+Returns:
+- timestamp: The timestamp of the UUID.
+*/
+time_v1 :: proc "contextless" (id: Identifier) -> (timestamp: time.Time) {
+	delta := cast(time.Duration)(raw_time_v1(id) - HNS_INTERVALS_BETWEEN_GREG_AND_UNIX) * 100
```

Comment:
This should probably just be a straight up `Time{_nsec=...}` because subtraction notwithstanding, we're converting a moment between epochs, not calculating an interval.